pyproject.toml

```diff
@@ -27,8 +27,8 @@ python = ">=3.8,<4.0"
 ecdsa = "*"
 bech32 = "*"
 requests = "*"
-protobuf = ">=3.19.4,<4"
-grpcio = "==1.47.0"
+protobuf = ">=4.21.6,<5.0dev"
```

Comment:
This range is problematic for the AEA framework, wouldn't play nice with other deps (`web3` if I remember correctly).

Reply:
Hmmm, this is sort of an issue then @5A11 . We can't really pin the dependencies to an old version for ever! We might need to pin the AEA version that is used for cosmpy.

Or at least we should evaluate if the latest web3 dependencies now play nicely with this version of AEA